docs/3.3-Common-Properties-Controls.md

```diff
@@ -0,0 +1,111 @@
+# Common Properties Element Types
+The editor determines the most appropriate control and panel type for each parameter based on the context (parameter type, role, group type, etc.). Although an author can specify a desired control for a given parameter via its `uihints`, this should be used sparingly and carefully - specifying an inappropriate control for a parameter will lead to undefined behavior.
+
+A visual documentation containing examples of the following group/panels and control types are available at https://wdp-common-canvas-dev.stage1.mybluemix.net/#/properties.
+
+## Group/Panel Controls
+Controls are grouped and arranged on panels and sub-panels within the "group_info" section of operator uihints. Some panels appear as tab controls, others are for managing shared dataset metadata, while still others contain nested sub-panels.
+
+### Group Types
+* `controls` A general panel type containing parameter controls.
+* `tabs` A tabbed control, each tab containing sub-panels and controls.
+* `subTabs` A horizontal sub-tabbed control, each tab containing sub-panels and controls. If displayed within a Tearsheet container, subtabs will be displayed vertically.
+* `panels` A panel type that contains sub-panels.
+* `panelSelector` A panel containing sub-panels that are shown or hidden based upon a controlling radio selection.
+* `columnSelection` A panel type containing field-selection controls that share a common set of fields.
+* `customPanel` A custom panel for displaying user defined control. See [3.5 Common Properties Custom Components](3.5-Common-Properties-Custom-Components.md#custom-panels) for more details.
+* `summaryPanel` A panel used in the flyout editor that will provide a link to open a wide flyout that can contain panels and parameter.
+* `actionPanel` A panel used for containing action controls.
+* `textPanel` A panel used to display static label and/or description. 
+* `twistyPanel` A panel used to display a panel title that expands to the panel content when clicked.
+* `columnPanel` A panel used to display children side by side. 
+* `tearsheetPanel` A tearsheet panel.  The panel can be opened/closed using the propertyController methods `setActiveTearsheet(groupId)` and `clearActiveTearsheet()`
+
+## Parameter Controls
+The following controls are supported in the Common Properties editor. Control types are intended for use with particular parameter types:
+
+### Control Types
```

Comment:
`slider` control documentation is not added in this section.

Reply:
> `slider` control documentation is not added in this section.

Hi @nmgokhale I have included slider control documentation now https://srikant-ch5.github.io/canvas/3.3-Common-Properties-Controls/#control-types . Previously I was taking https://github.com/elyra-ai/canvas/wiki as reference (slider is not included in this).